### DISCUSSION TITLE
How about adding a fixed textual representation to each star rating

### DISCUSSION THREAD

**Discussion by LokiMidgard** [bookwyrm-social/bookwyrm#3448] [Ideas]
The idea is to show a textual description when a star rating is chosen that describes how it should be interpreted.

**Motivation**

The interpretation of stars is somewhat depended of the context of the user.  A textual description could help to synchronize this between users.

E.g. for one user a good book he enjoyed would be 5 stars, because there was nothing to complain. For another this is equivalent with a 4 star rating, since he or she dose reserve 5 stars for books  that where true one of a kind works. It may be only 1 Star, but if you have only 5 stars one is still 20% difference

**suggestion for names**
1. Horrible
2. Bad
3. Average
4. Great
5. Masterpiece

---

I personally like the rating of myanimelist.net that I used as inspiration which has 10 Points with the descriptions:
 

1.  Appalling
2.  Horrible
3.  Very Bad
4.  Bad
5.  Average
6.  Fine
7.  Good
8.  Very Good
9.  Great
10.  Masterpiece

I find it very easy to rate those compared to stars.

**Comment by mouse-reeve** [comment]
We had a discussion about switching from a five star to a ten star system a while back, and compromised by allowing half stars. If you click a star a second time when making your rating, it becomes a half star. I hope that's an acceptable solution for you!

**Reply by LokiMidgard** [reply]
I have seen this issue, and my intention was not to go to an 10 point rating or replace the stars in any way.
It was more to add an additional text representation e.g. under the stars to let people have a more aligning understanding what 4 or 5 stars mean.
